### DISCUSSION TITLE
Finding bad rips

### DISCUSSION THREAD

**Discussion by DavidCWGA** [beetbox/beets#4440] [Q&A]
Is there a plugin, or an external tool, that can scan my library for music that was ripped from CD badly? Something like EAC's "AccurateRip", but after the fact.

I assume this would only work on lossless rips, but even that would be useful.

**Comment by RollingStar** [comment]
You'd first have to define what a "bad rip" is. I found this with: "detect skips in ripped audio" on google.

https://blog.magnatune.com/2013/09/ripcheck-detect-defects-in-cd-rips.html

https://www.discogs.com/forum/thread/218310

By no means an expert, but I think EAC works its magic by reading the CRC/integrity data from the CD. Once you rip it to WAV, that data is not saved. So you can't do that after the fact.

**Reply by DavidCWGA** [reply]
ripcheck looks really interesting, I'll give it a try, thanks
